README.md

```diff
@@ -40,6 +40,12 @@ yarn start
 
 This should start up a webserver at http://localhost:5000 which you can connect to.
 
+To run Jest tests for React Components, execute:
```

Comment:
Let's link to jest here:

`To run [Jest](https://jestjs.io/) tests for React components, execute:`